navigator/frontend/src/ui-core/src/index.ts

```diff
@@ -10,6 +10,7 @@ export { default as Link } from './Link';
 export { makeSidebarLink } from './SidebarLink';
 export { default as ParameterForm } from './ParameterForm';
 export { default as Popover } from './Popover';
+export { default as AgreementText } from './AgreementText';
```

Comment:
I don't see this file included in this PR.

Reply:
Thanks, as you noticed later, I initially had a full blown separate component in its own directory. I will go with your recommendation of a simple function, as it is more lightweight for this simple use case.